src/main/java/com/group24/decide/Utility.java

```diff
@@ -190,4 +190,28 @@ public static double findSmallestCircle(Datapoints a, Datapoints b, Datapoints c
         return radius;
 
     }
+
+    /**
+     * 
+     * Find the angle between three data points where one is the Vertex points,
+     * described as the vertexPoints in this method.
+     * 
+     * @param pointsA One datapoints
+     * @param vertexPoints The datapoints which angle we are calculating
+     * @param pointsB Second datapoints
+     * @return The angle of vertexPoints.
+     */
+    public static double calculateAngle(Datapoints pointsA, Datapoints vertexPoints, Datapoints pointsB){
+        
+        double b = calcEuclideanDistance(pointsA, vertexPoints);
+        double c = calcEuclideanDistance(pointsA, pointsB);
+        double a = calcEuclideanDistance(vertexPoints, pointsB);
+
+        // Law of cosinus a2=b2+c2−2bc⋅cosα
+        // α = aCos((b2 + c2 - a2) / 2bc)
+        
+        double angle = Math.acos((Math.pow(a, 2) +Math.pow(b, 2) - Math.pow(c, 2))/(2*b*a));
+
+        return angle;
+    }
 }
```

Comment:
Optional: I think it would be nice to also add a unit test for this function and maybe add to the description that it ouputs the angle in radians

Reply:
Yeah I agree, that would be clearer! Thank you for your input! 👍 I'll fix it!